src/app/ApplicationTemplate.Shared.Views/Behaviors/FormattingTextBoxBehavior.cs

```diff
@@ -195,7 +196,7 @@ private static void FormatText(TextBox textbox, string textFormat)
 			return;
 		}
 
-		_ = textbox.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
```

Comment:
Did you test that the behavior still works? (I guess it's used to automatically format a phone number, perhaps?)

Reply:
i did, it's working as expected :)

Reply:
Great! thanks :)